WebApplicationMVC/wwwroot/SingleView/index.html

```diff
@@ -503,10 +507,10 @@
             requestObject.typename = UIEventMessage;
             Post2Server(requestObject, '');
         }
-
-
+        
         $(function () {
-            openPage('Login');
+            let ClickedLink = localStorage.getItem('clickedLink');
+            openPage(ClickedLink);
```

Comment:
This was not the goal of the exercise. You were to launch a page from a new shortcut on the  'index.html' that launches a frame with the inputs that you store in a localstorage variable. Then you will add a new shortcut on the 'index.html' that will launch with the values being displayed. 2 frames.

Reply:
Do you want me to add a shortcut on Home\Index.
![Shortcut](https://github.com/wraptor66/Frameswork/assets/104307054/a6612ecd-000c-4cd9-8751-164b57910e13)
Steps i have to do

1. Add above shortcut on home screen (Home\Index) and add 2 applications there.
2. Application1 will have Form Details Frame.
3. Application2 will have Show Submitted form in other Frame.

Reply:
Please create wireframes and add images here.

Reply:
Hi Mitch , please review wireframes.
[Feature Story.docx](https://github.com/wraptor66/Frameswork/files/13460894/Feature.Story.docx)